sources/packages/backend/apps/workers/src/controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -45,7 +50,7 @@ export class ProgramInfoRequestController {
     try {
       const application = await this.applicationService.getApplicationById(
         job.variables.applicationId,
-        { loadDynamicData: false },
+        { loadDynamicData: true },
```

Comment:
Would it not be better if the program info request hash is created on application submission? 

And the worker just doing the comparison to decide the auto approval?

Reply:
I did not see a reason to, and preferred to have the hash generation at the worker.
Is there a reason to have it done at the submission? In general, whatever can be done later (after the submission) is preferable, IMO.